### PR TITLE
Add Dec 2017 Quarterly Update Changes

### PR DESCRIPTION
**Pull request by stryqx**
Add Exchange Server 2013 CU19 and Exchange Server 2016 CU8 files from December 2017 Quarterly Updates.

Fixes # .

Changes included in this PR:
-
-
-

### CODE REVIEW

File: install.ps1 [hunk 1]
```diff
@@ -108,6 +112,8 @@ if (($exchserver.admindisplayversion).major -eq 8 -and ($exchserver.admindisplay
 	$EXVER="Exchange 2016 CU6"
 } elseif (($exchserver.admindisplayversion).major -eq 15 -and ($exchserver.admindisplayversion).minor -eq 1 -and ($exchserver.admindisplayversion).build -eq 1261) {
 	$EXVER="Exchange 2016 CU7"
+} elseif (($exchserver.admindisplayversion).major -eq 15 -and ($exchserver.admindisplayversion).minor -eq 1 -and ($exchserver.admindisplayversion).build -eq 1415) {
+	$EXVER="Exchange 2016 CU7"
```
Comment (btric-development) [inline install.ps1]: Should this be $EXVER="Exchange 2016 CU8"

Comment (stryqx) [reply install.ps1]: Yes it should. Thanks for spotting that.

Comment (stryqx) [reply install.ps1]: Fixed a typo in README.md as well.

**Comment by aledr** [comment]
Any due date for this?

**Comment by SamenwerkingsverbandWesterkwartier** [comment]
Hi, Can you tell when the support for Exchange 2016 CU8 is going to be released?

**Comment by JelmerT87** [comment]
Hi, Can you tell me when the support for Exchange 2016 cu8 going to be released?

**Comment by SamenwerkingsverbandWesterkwartier** [comment]
when i try to download the new release for the 2016 cu8 version i receive the old kimsigner version. Is that correct?

**Comment by Pro** [comment]
The current release is still v3.0.12

v3.0.13 will include the support for 2016 cu8. Before releasing it, I want to wait for #192

**Comment by Pro** [comment]
3.0.13 is now released:
https://github.com/Pro/dkim-exchange/releases/tag/v3.0.13

**Pull request merged**